templates/drupal-11/.github/workflows/build_release.yml

```
@@ -0,0 +1,48 @@
+on:
+  push:
+    tags:
+      - "*.*.*"
+
+name: Create Github Release
+
+permissions:
+  contents: write
+
+env:
+  COMPOSE_USER: runner
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    env:
+      COMPOSER_ALLOW_SUPERUSER: 1
+      APP_ENV: prod
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Composer install
+        run: |
+          docker network create frontend
+          docker compose run --rm --user=root phpfpm composer install --no-dev -o --classmap-authoritative
```

`--classmap-authoritative` implicitly enables `--optimize-autoloader` (`-o`):

```suggestion
          docker compose run --rm --user=root phpfpm composer install --no-dev --classmap-authoritative
```

We've had issues with using `--classmap-authoritative` in Drupal projets (resulting in what's described in https://www.drupal.org/project/drupal/issues/3334213), but I don't know if this has been resolved in Drupal 11. In (most of) our deployment scripts for Drupal we have an incantation effectively undoing what `--classmap-authoritative` does, e.g. https://github.com/itk-dev/ai-screening/blob/develop/.woodpecker/prod.yml#L27.